### ISSUE TITLE
Add dashboard support for notification click action

### ISSUE THREAD

**Issue by skyw33** [home-assistant/android#552]
**Is your feature request related to a problem? Please describe.**
When I call the notify service with either of the following data scenarios, it opens the app, but does not take me to the correct dashboard/view.
```

data:
  actions:
    - action: URI
      title: View Live
      uri: /live-cameras/front-porch
  image: 'https://homeassistant.local:8123/local/doorbellpress.jpg'
  tag: doorbell
message: Doorbell Press


```

```
data:
  clickAction: /live-cameras/front-porch
  image: 'https://homeassistant.local:8123/local/doorbellpress.jpg'
  tag: doorbell
message: Doorbell Press

```

where /live-cameras/ is a non-default dashboard.

**Describe the solution you'd like**
Clicking on the action should take me to the correct dashboard.

**Describe alternatives you've considered, if any**
If I specify the full http:// address, it opens the correct dashboard, but takes me to a browser, not the home assistant app.

**Additional context**

**Comment by kmlucy** [comment]
I'd like to add to this request the ability to open other relative urls aside from other Lovelace dashboards. For instance, `/developer-tools/logs` or `/hacs/installed`.

**Comment by terencevs** [comment]
Not sure if this would be the correct feature for this, but i would like to be able to send a push notification that appears in the notification panel as well. (I current achieve this by sending push and persistent notification but they don't link)

When you click on the push notification it takes you to the notification in the system.

**Comment by shbatm** [comment]
@terencevs See #398 .

**Comment by kmlucy** [comment]
This looks like it was fixed for me in #612.

**Comment by dshokouhi** [comment]
Working for me as well, closing this request now. Thanks for pointing it out @kmlucy
